src/main.js

```diff
@@ -49,8 +49,13 @@ module.exports.apifyGoogleAuth = async ({ scope, tokensStore, credentials, googl
     const authorizeUrl = oAuth2Client.generateAuthUrl({
         access_type: 'offline',
         scope: `https://www.googleapis.com/auth/${scope}`,
+        // Always prompt for user consent otherwise, there is a chance to not obtain refresh_token
+        // https://stackoverflow.com/a/10857806
+        prompt: 'consent'
```

Comment:
As mentioned in the stackoverflow - It helps in development when you are testing/using the same account for the same app and redirectUrl. The refresh_token seems to be returned only for the first time in this case. Then you have to manually go to your google account and remove the consent and then give it again to the application I noticed this because otherwise I had to give new consent every 30minutes since the credentials were saved without `refresh_token` which is required for obtaining the new tokens (which is pretty annoying).

See https://github.com/googleapis/google-api-nodejs-client/issues/750#issuecomment-304521450

Reply:
you are not storing the `code`, you are storing the received tokens.